build-system/tasks/e2e/index.js

```diff
@@ -25,14 +25,14 @@ const glob = require('glob');
 const http = require('http');
 const log = require('fancy-log');
 const Mocha = require('mocha');
+const opn = require('opn');
```

Comment:
Can we switch this to `open` instead of `opn`, which has been [deprecated](https://www.npmjs.com/package/opn)?

(Future PR is okay.)

Reply:
Yes, see PR description (not doing it in this PR because it will require updating uses throughout, and this will unblock the build)

Reply:
https://github.com/ampproject/amphtml/pull/30459